src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/generate.cmd

```diff
@@ -6,7 +6,7 @@
 @echo off
 set autoRestVersion=0.15.0-Nightly20160212
 if  "%1" == "" (
-    set specFile="ComputeCompositeClient.json"
+    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-06-15/swagger/compositeComputeClient.json"
```

Comment:
I think it's OK if you think it's better to refer to the file by link, but 1) the generator shall support using local file as well. In some cases, a build system is not supposed to read any files from github.com or any external links; and 2) this type of changes shall be applied to all other specs in the project.